media/pages/sk/podmienky-pouzivania/page.html

```diff
@@ -4,10 +4,10 @@
 <h1>Podmienky používania portálu chcemvedieť.sk</h1>
  
 
-<p>
+<h3>
```

Comment:
@martinmacko47 Iba taky detail, na funkcnosti bez zmeny. V celom subore je v `<p>` tagu na konci riadku ak text pokracuje medzera. Aj zalomenie by som este spravil na 100 znakov. Mam to upravid alebo nechat tak?

Reply:
Kludne to mozes zformatovat. Povyhadzovat vsetky trailing medzery a text zalomit na 100 znakov (alebo neviem ako sirku zvykneme pouzivat inde). Akurat pls v separatnom PR.